pkg/utils/client/apply.go

```diff
@@ -283,3 +318,35 @@ func newUnstructuredFor(obj runtime.Object) *unstructured.Unstructured {
 
 	return u
 }
+
+func (a *Applier) restClientFor(gv schema.GroupVersion) (*rest.RESTClient, error) {
+	restConfig, err := a.configFor(gv)
+	if err != nil {
+		return nil, fmt.Errorf("error constructing config for Group Version %+v: %v", gv, err)
```

Comment:
The error below uses "failed to ..." while this uses "error ...", perhaps we should be consistent?